.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: pnpm/action-setup@v2.2.1
         with:
-          version: 6.23.2
+          version: 6.32.11
```

Comment:
We can also omit this as `pnpm/action-setup` reads `packageManager` too.

Reply:
https://github.com/pnpm/action-setup/blob/57b9359b4c69c79ca51422d02ca739a7957f4aed/src/install-pnpm/run.ts#L25-L27 

this is not great. basically curlbashing a hardcoded version of pnpm to then install another version? I don't get it.

what about this?
read packageManager from package.json and send that to  `npm i -g`
```bash
PNPM_VER=$(jq -r '.packageManager | if .[0:5] == "pnpm@" then .[5:] else "packageManager in package.json does not start with pnpm@\n" | halt_error(1)  end' package.json)
npm i -g pnpm@$PNPM_VER
```

that would exit with 1 if packageManager isn't set or doesn't start with pnpm@

Reply:
I removed the version for now and will see if we can get the issue fixed in `action-setup`: https://github.com/pnpm/action-setup/issues/38